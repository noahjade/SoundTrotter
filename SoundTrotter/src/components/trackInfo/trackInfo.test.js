import renderer from "react-test-renderer";
import TrackInfo from ".";

test("Correctly renders track info", () => {
  const mockTrackInfo = {
    album: {
      images: [
        {
          url: "image-url",
        },
      ],
      name: "album",
    },
    name: "track",
    artists: [
      {
        name: "artist1",
      },
      {
        name: "artist2",
      },
      {
        name: "artist3",
      },
    ],
  };

  const view = renderer.create(<TrackInfo trackItem={mockTrackInfo} />);
  expect(view).toMatchSnapshot();
});
