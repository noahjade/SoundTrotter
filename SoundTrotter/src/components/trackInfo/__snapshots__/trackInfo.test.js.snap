// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correctly renders track info 1`] = `
Array [
  <div
    className="image"
  >
    <img
      alt="album"
      src="image-url"
    />
  </div>,
  <div
    className="info"
  >
    <span
      className="info__title"
    >
      track
    </span>
    <span
      className="info__artists"
    >
      artist1, artist2, artist3
    </span>
  </div>,
]
`;
